<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWidget</class>
 <widget class="QWidget" name="BrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="m_browserStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="GameBrowser" name="m_gameBrowser"/>
     <widget class="StreamBrowser" name="m_streamBrowser"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameBrowser</class>
   <extends>QWidget</extends>
   <header>AYAYA/browser/gamebrowser.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StreamBrowser</class>
   <extends>QWidget</extends>
   <header>AYAYA/browser/streambrowser.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
